src/main/kotlin/net/atos/zac/smartdocuments/rest/SmartDocumentsRESTService.kt

```diff
@@ -0,0 +1,44 @@
+package net.atos.zac.smartdocuments.rest
+
+import jakarta.enterprise.inject.Instance
+import jakarta.inject.Inject
+import jakarta.inject.Singleton
+import jakarta.ws.rs.Consumes
+import jakarta.ws.rs.GET
+import jakarta.ws.rs.POST
+import jakarta.ws.rs.Path
+import jakarta.ws.rs.Produces
+import jakarta.ws.rs.core.MediaType
+import net.atos.zac.authentication.LoggedInUser
+import net.atos.zac.authentication.SecurityUtil
+import net.atos.zac.policy.PolicyService
+import net.atos.zac.smartdocuments.SmartDocumentsService
+import net.atos.zac.smartdocuments.templates.SmartDocumentsTemplateConverter.toModel
+import nl.lifely.zac.util.AllOpen
+import nl.lifely.zac.util.NoArgConstructor
+
+@Singleton
+@Path("smartdocuments")
+@Consumes(MediaType.APPLICATION_JSON)
+@Produces(MediaType.APPLICATION_JSON)
+@NoArgConstructor
+@AllOpen
+class SmartDocumentsRESTService @Inject constructor(
+    private val smartDocumentsService: SmartDocumentsService,
+    private val loggedInUserInstance: Instance<LoggedInUser>
+) {
+
+    @GET
+    @Path("templates")
```

Comment:
This REST service is not about templates as such right? It is about the zaakparameter-templates mapping or something like that right? it should probably have to getters: one to retrieve all current template mappings for the provided zaakparameter config and one to retrieve all SD templates. And then one update/post method to update those mappings. Something like that?